packages/frontend-2/components/settings/user/profile/EditAvatar.vue

```diff
@@ -25,7 +25,7 @@ graphql(`
 `)
 
 const props = defineProps<{
-  user: UserProfileEditDialogAvatar_UserFragment
+  user: User
```

Comment:
You're asking for way too much when requiring the full User type, making it practically impossible to actually use these components without overriding/ignoring TS errors, use fragments to define specific requirements -
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#d40e40efc32f4e5ea67b68e848aadcd0